tests/test_mypy.py

```diff
@@ -1,21 +1,44 @@
 import pytest
+from django.db.models import F, Func
 from mypy import api
 
 from .test_decorator import OPTIONS
 
 
+def safe_repr(value):
```

Comment:
Nice work! Today I learned something new :)

I like the solution; it's clever, but maybe too clever in a way that it will be hard for others to understand what's going on. I am a bit on a fence, but maybe we should just be explicit and redefine everything instead of importing the `OPTIONS`.

```python
OPTIONS = [
    ('admin_order_field', '"radius"'),
    ('admin_order_field', 'Lower(F("person_name"))'),
    ('boolean', 'True'),
    ('empty_value_display', '"Undefined"'),
    ('short_description', '"Is big?"'),
]
if django.VERSION[:2] <= (1, 11):
    OPTIONS.append(('allow_tags', "True"))
```

I know you put a lot of effort into solving this issue, so I am curious to hear what you think about this.